reactor-core/src/test/java/reactor/core/publisher/FluxFilterFuseableTest.java

```diff
@@ -266,10 +278,19 @@ public void discardConditionalTryOnNextPredicateMiss() {
 
 	@Test
 	public void discardConditionalPollAsyncPredicateFail() {
+		CountDownLatch latch = new CountDownLatch(10);
 		StepVerifier.create(Flux.range(1, 10) //range uses tryOnNext, so let's use just instead
 		                        .publishOn(Schedulers.newSingle("discardPollAsync"))
 		                        .filter(i -> { throw new IllegalStateException("boom"); })
 		                        .filter(i -> true)
+		                        .contextWrite(previous -> {
```

Comment:
does this change really relates to this PR?

Reply:
This fails on my machine all the time (m1 does goos job in detecting such things)